server/views/pages/search.njk

```diff
@@ -1,25 +1,19 @@
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
 {% from "govuk/components/date-input/macro.njk" import govukDateInput %}
 {%- from "govuk/components/error-summary/macro.njk" import govukErrorSummary -%}
 {% from "govuk/components/input/macro.njk" import govukInput %}
 {% from "govuk/components/radios/macro.njk" import govukRadios %}
 {% from "govuk/components/table/macro.njk" import govukTable %}
 {%- from "govuk/components/tag/macro.njk" import govukTag -%}
+{% from "govuk/components/select/macro.njk" import govukSelect %}
 {% from "moj/components/filter/macro.njk" import mojFilter %}
 {%- from "moj/components/pagination/macro.njk" import mojPagination -%}
 {% from "../partials/multiselectfilter/macro.njk" import multiSelectFilter %}
-{% from "govuk/components/button/macro.njk" import govukButton %}
-{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
-{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
-{% from "govuk/components/input/macro.njk" import govukInput %}
-{% from "govuk/components/radios/macro.njk" import govukRadios %}
-{% from "govuk/components/select/macro.njk" import govukSelect %}
-{% from "govuk/components/table/macro.njk" import govukTable %}
-{% from "govuk/components/tag/macro.njk" import govukTag %}
-{% from "moj/components/filter/macro.njk" import mojFilter %}
 
 {% extends "../partials/layout.njk" %}
 
-{% set pageTitle = "Prisoner Search" %}
+{% set pageTitle = "HPA Prisoner Search" %}
```

Comment:
probably wouldn't bother with the `HPA` here - I'm not a great fan of the word `Application` in this project anyway

Reply:
ha ha - was just replicating the current